subsys/bluetooth/Kconfig.iso

```diff
@@ -67,7 +67,6 @@ if BT_ISO
 config BT_ISO_MAX_CHAN
 	int "Maximum number of simultaneous ISO channels"
 	depends on BT_ISO
-	default BT_MAX_CONN if BT_CONN
```

Comment:
It is  requirement that `BT_ISO_MAX_CHAN` be more than `BT_MAX_CONN` otherwise `CONFIG_BT_AUDIO_UNICAST_CLIENT_GROUP_STREAM_COUNT=2` in the unicast_client test fails to build.

https://github.com/zephyrproject-rtos/zephyr/pull/52842/files#diff-28c0079e66f43f2f867cb0221d31158c38e850327702755b4abc12afe90c4e50L26

Reply:
If a test case require more than the default, we should just increase the value for the test case; we shouldn't modify the default based on a test IMO

Reply:
ok... something has changed after rebase, this change is no longer required to build the unicast_client test.